ot3_state_manager/ot3_state_manager/ot3_state_manager.py

```diff
@@ -0,0 +1,209 @@
+        left_pipette_model: Optional[PipetteModel],
+        right_pipette_model: Optional[PipetteModel],
+        use_gripper: bool,
+    ) -> OT3StateManager:
+        """Helper function to build an OT3StateManager object.
+
+        Takes optional left pipette and right pipette model names. Passing None for the
+        pipette models names tells the system that there is no pipette attached that
+        slot.
+
+        Pass True for use_gripper parameter to tell the model there is a gripper
+        attached. Pass False if no gripper is attached.
+
+        If you are attaching a high-throughput pipette it must go in the right-side
+        slot.
+        """
+        left_pipette = (
+            LeftPipette(model=left_pipette_model)
+            if left_pipette_model is not None
+            else None
+        )
+        right_pipette = (
+            RightPipette(model=right_pipette_model)
+            if right_pipette_model is not None
+            else None
+        )
+        gripper = Gripper() if use_gripper else None
+
+        return cls(
+            left_pipette=left_pipette, right_pipette=right_pipette, gripper=gripper
+        )
+
+    def __init__(
+        self,
+        left_pipette: Optional[LeftPipette],
+        right_pipette: Optional[RightPipette],
+        gripper: Optional[Gripper],
+    ) -> None:
+        """Creates OT3StateManager object.
+
+        Args:
+            left_pipette: PipetteModel to attach, or None.
+            right_pipette: PipetteModel to attach, or None.
+            gripper: Whether or not to attach a gripper.
+        """
+        if (
+            left_pipette is not None
+            and left_pipette.model == PipetteModel.MULTI_96_1000
+        ):
+            raise ValueError(
+                "High-throughput pipette must be connected to right pipette mount"
+            )
+
+        if (
+            left_pipette is not None
+            and right_pipette is not None
+            and right_pipette.model == PipetteModel.MULTI_96_1000
+        ):
+            raise ValueError(
+                "When using high-throughput pipette you cannot have a pipette in the "
+                "left mount"
+            )
+
+        self.head = Head()
+        self.gantry_x = GantryX()
+        self.gantry_y = GantryY()
+        self.left_pipette = left_pipette
+        self.right_pipette = right_pipette
+        self.gripper = gripper
+
+    def _pos_dict(self) -> Dict[OT3Axis, Optional[Position]]:
+        """Generates lookup dict for class's position properties."""
+        left_pipette_pos = (
+            self.left_pipette.position if self.left_pipette is not None else None
+        )
+        right_pipette_pos = (
+            self.right_pipette.position if self.right_pipette is not None else None
+        )
+        gripper_pos = self.gripper.position if self.gripper is not None else None
+
+        return {
+            Head.gripper_mount_position.axis: self.head.gripper_mount_position,
```

Comment:
A little confused as to what this gripper mount position is. I think when @sfoster1 and I discussed awhile back, we decided on not characterizing the gripper mount as an axis thing because it doesn't actually move. Is this meant to be the Z axis of the gripper itself?

If so, would it be possible to call this `gripper_z_position`?

Reply:
Addressed in: 
https://github.com/Opentrons/ot3-firmware/pull/418/commits/df794f19371acc636beebcc2b974988975faf588
https://github.com/Opentrons/ot3-firmware/pull/418/commits/ed2860dd90e8aa7f14d009ec625d81147a1c404d